packages/product/integration-tests/__tests__/services/product-module-service/products.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { MedusaModule } from "@medusajs/modules-sdk"
```

Comment:
suggestion: should we rename that file `product-update.spec.ts`? Because if we move the create such as your suggestion we will have conflict right?

Reply:
No, the create should be in this file too. I just didn't move it because the base PR isn't merged yet.

Reply:
the base pr has been merge this week end :) also, in that case put all the update test in a separate describe

Reply:
It is in a separate describe